examples/io.birb

```diff
@@ -0,0 +1,6 @@
+grab("birb:io");
+
+scremLn(cwd.runtimeType); // output String
+scremLn(system("echo", ["Henlo"], true).runtimeType); // output Map
```

Comment:
This can be more meaningful, and cover more of the code. Maybe output the stdout, stderr and exitCode values or iterate through stderr and stdout? I'm not sure but it definitely needs improvement.